src/iop/temperature.c

```diff
@@ -198,55 +207,28 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_RAW;
 }
 
-static gboolean _set_preset_camera(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
-                                   GdkModifierType modifier, gpointer data)
-{
-  dt_iop_module_t *self = data;
-  dt_iop_temperature_gui_data_t *g = self->gui_data;
-  dt_bauhaus_combobox_set(g->presets, DT_IOP_TEMP_AS_SHOT);
-  return TRUE;
-}
-
-static gboolean _set_preset_camera_neutral(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
-                                           GdkModifierType modifier, gpointer data)
-{
-  dt_iop_module_t *self = data;
-  dt_iop_temperature_gui_data_t *g = self->gui_data;
-  dt_bauhaus_combobox_set(g->presets, DT_IOP_TEMP_D65);
-  return TRUE;
-}
-
 static gboolean _set_preset_spot(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
-                                 GdkModifierType modifier, gpointer data)
+                                 GdkModifierType modifier, dt_iop_module_t *self)
 {
-  dt_iop_module_t *self = data;
   dt_iop_temperature_gui_data_t *g = self->gui_data;
   dt_bauhaus_combobox_set(g->presets, DT_IOP_TEMP_SPOT);
   return TRUE;
 }
 
-static gboolean _set_preset_user(GtkAccelGroup *accel_group, GObject *acceleratable, guint keyval,
-                                 GdkModifierType modifier, gpointer data)
-{
-  dt_iop_module_t *self = data;
-  dt_iop_temperature_gui_data_t *g = self->gui_data;
-  dt_bauhaus_combobox_set(g->presets, DT_IOP_TEMP_USER);
-  return TRUE;
-}
-
 void init_key_accels(dt_iop_module_so_t *self)
 {
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "tint"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "temperature"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "red"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "green"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "blue"));
+  dt_accel_register_slider_iop(self, FALSE, NC_("accel", "emerald"));
   dt_accel_register_combobox_iop(self, FALSE, NC_("accel", "presets"));
```

Comment:
since combo is now labeled "settings" can we switch this up too? i forgot about that in mine ;)